q2cli/builtin/dev.py

```diff
@@ -29,3 +30,90 @@ def dev():
 def refresh_cache():
     import q2cli.core.cache
     q2cli.core.cache.CACHE.refresh()
+
+
+install_theme_help = \
+    ("Allows for customization of q2cli's command line styling based on an "
+     "imported .ini file. If you are unfamiliar with the format of .ini "
+     "files look here https://en.wikipedia.org/wiki/INI_file."
+     "\n"
+     "\n"
+     "The .ini allows you to customize text on the basis of what that text "
+     "represents with the following supported text types: command, options, "
+     "type, default_arg, required, emphasis, problem, errors, and success. "
+     "These will be your headers in the '[]' brackets. "
+     "\n"
+     "\n"
+     "Command refers to the name of the command you issued. Option refers "
+     "to the arguments you give to the command when running it. Type refers "
+     "to the Qiime2 semantic typing of these arguments (where applicable). "
+     "Default_arg refers to the label next to the argument indicating its "
+     "default value (where applicable), and if it is required (where "
+     "applicable). Required refers to any arguments that must be passed to "
+     "the command for it to work and gives them special formatting on top of "
+     "your normal 'options' formatting. Emphasis refers to any emphasized "
+     "pieces of text within help text. Problem refers to the text informing "
+     "you there were issues with the command. Error refers to the text "
+     "specifying those issues. Success refers to text indicating a process "
+     "completed as expected."
+     "\n"
+     "\n"
+     "The following pieces of the text's formatting may be customized: bold, "
+     "dim (if true the text's brightness is reduced), underline, blink, "
+     "reverse (if true foreground and background colors are reversed), and "
+     "finally fg (foreground color) and bg (background color). The first five "
+     "may each be either true or false, while the colors may be set to any of "
+     "the following: black, red, green, yellow, blue, magenta, cyan, white, "
+     "bright_black, bright_red, bright_green, bright_yellow, bright_blue, "
+     "bright_magenta, bright_cyan, or bright_white.")
+
+
+@dev.command(name='install-theme',
+             short_help='Install new command line theme.',
+             help=install_theme_help,
+             cls=ToolCommand)
+@click.option('--theme', required=True,
+              type=click.Path(exists=True, file_okay=True,
+                              dir_okay=False, readable=True),
+              help='Path to file containing new theme info')
+def install_theme(theme):
+    import os
+    import shutil
+    import q2cli.util
+    from q2cli.core.config import CONFIG
+    CONFIG.parse_file(theme)
```

Comment:
It would be nice to use `q2cli.util.exit_with_error` if something did go wrong (this creates a header and makes things red).

You can check out `qiime tools import` for an example of usage.

Reply:
If it is one of your expected validation errors, we'd want to suppress the stack trace, and anything else, we probably want to print it.

Reply:
Flags can't be made required in the same way as non-flag parameters, so I did this 
```
    if not yes:
        e = ValueError("Missing required '--yes' flag.")
        header = "Missing required flag '--yes'"
        q2cli.util.exit_with_error(e, header=header, traceback=None)
```
But I don't know what to make the messages. They probably shouldn't just be the same thing like they are now

Reply:
@Oddant1, would it make sense to compress this slightly? eg

```python
if not yes:
    message = "Missing required flag '--yes'"
    q2cli.util.exit_with_error(ValueError, message=message, traceback=False)
```

This would then allow `q2cli.util.exit_with_error` to construct an exception, deal with a traceback should one be desired, etc?